app/assets/stylesheets/common/components/tap-tile.scss

```diff
@@ -0,0 +1,39 @@
+.tap-tile-grid {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+  .tap-tile {
+    padding: 10px 5px;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    align-items: center;
+    border: 2px solid #c7d2db;
+    margin: 0 0 20px;
+    width: 28%;
+    font-size: 12px;
```

Comment:
look at using $font-size

Reply:
For posterity this was referring to font-0, font-down-*, and font-up-* in `app/assets/stylesheets/common/foundation/variables.scss`